src/Ledger.Net.Tests/LedgerTests.cs

```diff
@@ -17,26 +17,40 @@ public class LedgerTests
 
         static readonly UsageSpecification[] _UsageSpecification = new[] { new UsageSpecification(0xffa0, 0x01) };
 
-        [Fact]
-        public async Task GetAddressAnyBitcoinApp()
-        {
-            var ledgerManager = await GetLedger();
+        //[Fact]
+        //public async Task GetAddressAnyBitcoinApp()
+        //{
+        //    var ledgerManager = await GetLedger();
 
-            await ledgerManager.SetCoinNumber();
+        //    await ledgerManager.SetCoinNumber();
 
-            var address = await ledgerManager.GetAddressAsync(0, false, 0, false);
-        }
+        //    var address = await ledgerManager.GetAddressAsync(0, false, 0, false);
+        //}
+
+        //[Fact]
+        //public async Task GetAddress()
+        //{
+        //    var ledgerManager = await GetLedger();
+
+        //    var address = await ledgerManager.GetAddressAsync(0, false, 0, false);
+        //    if (address == null)
+        //    {
+        //        throw new Exception("Address not returned");
+        //    }
+        //}
 
         [Fact]
-        public async Task GetAddress()
+        public async Task SignEthereumTransaction()
         {
             var ledgerManager = await GetLedger();
+            ledgerManager.SetCoinNumber(60);
 
-            var address = await ledgerManager.GetAddressAsync(0, false, 0, false);
-            if (address == null)
-            {
-                throw new Exception("Address not returned");
-            }
+            byte[] rlpEncodedTransactionData = { 227, 128, 132, 59, 154, 202, 0, 130, 82, 8, 148, 139, 6, 158, 207, 123, 242, 48, 225, 83, 184, 237, 144, 59, 171, 242, 68, 3, 204, 162, 3, 128, 128, 4, 128, 128 };
```

Comment:
This is the RLP encoded transaction data which contains all data required to send a transaction like the nonce, gas limit, gas price, send address, value, etc.

This was encoded using the [Nethereum Transaction class](https://github.com/Nethereum/Nethereum/blob/master/src/Nethereum.Signer/Transaction.cs). Since RLP encoding is quite complicated, it's best not to create a new implementation inside Ledger.Net. 

A potential solution would be to simply pass in the RLP encoded transaction data into the ```ledgerManager.SignTransactionAsync``` method. This way, for other cryptocurrencies, the transaction byte data can be handled outside the Ledger.Net library, and then passed through the method. 

As a potential solution, it may not be the most simple to use, however it would be quite modular.